meinberlin/apps/users/templatetags/userindicator.py

```diff
@@ -26,5 +26,6 @@ def get_next_url(request):
 
 @register.inclusion_tag('meinberlin_users/indicator.html', takes_context=True)
 def userindicator(context):
-    context['redirect_field_value'] = get_next_url(context['request'])
+    if hasattr(context, 'request'):
+        context['redirect_field_value'] = get_next_url(context['request'])
```

Comment:
Hmm, I don't get that. For me, the error page is already shown without that and also the Anmelden/Registrieren button is shown with or without that.

(I needed so long to get the error pages run locally, we should document that somewhere...
- set DEBUG=False and add ALLOWED_HOSTS = ['localhost'] in/to settings/dev.py
- start server with: python manage.py runserver --insecure
)